pkgs/development/libraries/qt-4.x/4.8/dlopen-webkit-nsplugin.diff

```diff
@@ -20,7 +20,7 @@ index 2fe69d1..b658e4a 100644
      // The code below has the same effect as this one:
      // Display *gdkDisplay = gdk_x11_display_get_xdisplay(gdk_display_get_default());
 -    QLibrary library(QLatin1String("libgdk-x11-2.0"), 0);
-+    QLibrary library(QLatin1String("@gdk_pixbuf@/lib/libgdk-x11-2.0"), 0);
++    QLibrary library(QLatin1String("@gtk@/lib/libgdk-x11-2.0"), 0);
```

Comment:
You should remove

https://github.com/NixOS/nixpkgs/blob/c29b17eecbeb0798ff3148c3a23978261f3348df/pkgs/development/libraries/qt-4.x/4.8/default.nix#L103